### PR TITLE
HTCONDOR-1697 Add :macro: tags to config knobs in manual

### PR DESCRIPTION
**Pull request by GregThain**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [x] Check for documentation, if needed
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/admin-manual/monitoring.rst [hunk 1]
```diff
@@ -273,9 +262,9 @@ invalidated ClassAd. An example of a ClassAd that could benefit from
 being absent is a system with an uninterruptible power supply that shuts
 down cleanly but unexpectedly as a result of a power outage. To cause
 all invalidated ClassAds to become absent instead of invalidated, set
-``EXPIRE_INVALIDATED_ADS`` :index:`EXPIRE_INVALIDATED_ADS` to
+:macro:`EXPIRE_INVALIDATED_ADS` to
 ``True``. Invalidated ClassAds will instead be treated as if they
-expired, including when evaluating ``ABSENT_REQUIREMENTS``.
+expired, including when evaluating macro:`ABSENT_REQUIREMENTS`.
```
Comment (ColeBollig) [inline docs/admin-manual/monitoring.rst:267]:
```suggestion
expired, including when evaluating :macro:`ABSENT_REQUIREMENTS`.
```

File: docs/admin-manual/policy-configuration.rst [hunk 2]
```diff
@@ -2619,10 +2615,10 @@ and that it is partitionable.
 
 Enable use of the *condor_negotiator*-side resource consumption policy,
 allocating the job-requested number of cores to the dynamic slot, and
-use ``SLOT_WEIGHT`` :index:`SLOT_WEIGHT` to assess the user usage
+use :macro:`SLOT_WEIGHT` to assess the user usage
 that will affect user priority by the number of cores allocated. Note
-that the only attributes valid within the ``SLOT_WEIGHT``
-:index:`SLOT_WEIGHT` expression are Cpus, Memory, and disk. This
+that the only attributes valid within the 
+:ndex:`SLOT_WEIGHT` expression are Cpus, Memory, and disk. This
```
Comment (ColeBollig) [inline docs/admin-manual/policy-configuration.rst:2621]:
```suggestion
:macro:`SLOT_WEIGHT` expression are Cpus, Memory, and disk. This
```

File: docs/admin-manual/security.rst [hunk 3]
```diff
@@ -212,14 +212,14 @@ If the condor daemon receiving the connection is the *condor_collector*, it firs
...
 *condor_startd* daemon), it will check ``ALLOW_ADVERTISE_STARTD``.  And if the
 incoming command is for a *condor_master* (which runs on all HTCondor nodes) it
-will check ``ALLOW_ADVERTISE_MASTER``.  If the list it checks is undefined, it will
-then check ``ALLOW_DAEMON`` instead.
+will check :macro:`ALLOW_ADVERTISE_MASTER`.  If the list it checks is undefined, it will
+then check :macro:`ALLOW_DAEMON` instead.
 
 If the condor daemon receiving the connection is not a *condor_collector*, the
-``ALLOW_DAEMON`` is the only list that is looked at.
+:macro:`ALLOW_DAEMON` is the only list that is looked at.
```
Comment (ColeBollig) [inline docs/admin-manual/security.rst:222]: I am unsure if the various ALLOW_* config options actually exist in the configuration macro docs

File: docs/admin-manual/security.rst [hunk 4]
```diff
@@ -212,14 +212,14 @@ If the condor daemon receiving the connection is the *condor_collector*, it firs
 checks to see if there are specific authorization lists for daemons advertising
 to the collector (i.e. joining the pool).  If the incoming command is
 advertising a submit node (i.e. a *condor_schedd* daemon), it will check
-``ALLOW_ADVERTISE_SCHEDD``.  If the incoming command is for an execute node (a
+:macro:`ALLOW_ADVERTISE_SCHEDD`.  If the incoming command is for an execute node (a
 *condor_startd* daemon), it will check ``ALLOW_ADVERTISE_STARTD``.  And if the
```
[the remark this answers is not in the document]
Comment (GregThain) [reply docs/admin-manual/security.rst:216]: Yes, but we'll fix that later...

File: docs/admin-manual/security.rst (hunk 3 above)
Comment (GregThain) [reply docs/admin-manual/security.rst:222]: The don't, but that's a problem for another day.

File: docs/admin-manual/networking.rst [hunk 5]
```diff
@@ -785,17 +782,17 @@ Running HTCondor on an IPv6 Network Stack
...
+To require IPv4, you may set :macro:`ENABLE_IPV4`
 to true; if the machine does not have an interface with an IPv4 address,
 HTCondor will not start. Likewise, to require IPv6, you may set
-``ENABLE_IPV6`` :index:`ENABLE_IPV6` to true.
+:macro:`ENABLE_IPV6` to true.
 
-If you set ``ENABLE_IPV4`` :index:`ENABLE_IPV4` to false, HTCondor
+If you set :macro:`ENABLE_IPV4` to false, HTCondor
 will not use IPv4, even if it is available; likewise for ``ENABLE_IPV6``
 :index:`ENABLE_IPV6` and IPv6.
```
[the remark this answers is not in the document]
Comment (GregThain) [reply docs/admin-manual/networking.rst]: pushed change.

**Comment by GregThain** [comment]
Changes look good.  Let's go!

**Pull request merged**